<launch>

  <!-- launch the planner -->
  <include file="$(find bwi_coffee)/launch/planner.launch">
    <arg name="dry_run" value="true" />
    <arg name="use_speech" value="true" />
    <arg name="rigid_knowledge_file" value="$(find bwi_coffee)/domain/real_facts.asp" />
    <arg name="initial_file" value="$(find bwi_coffee)/domain/real_initial.asp" />
  </include>

  <!-- launch the gui -->
  <node name="segbot_gui" pkg="segbot_gui" type="question_dialog_plugin" />

  <!-- launch the sound play node -->
  <node name="soundplay" pkg="sound_play" type="soundplay_node.py" />

</launch>
